src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -65,6 +58,45 @@ class IOUAmountPage extends React.Component {
         }
     }
 
+    /**
+     * Update amount with number or Backspace pressed.
+     * Validate new amount with decimal number regex up to 6 digits and 2 decimal digit to enable Next button
+     *
+     * @param {String} key
+     * @param {String} event
+     */
+    onlyAllowNumericValues(key, event) {
```

Comment:
This function name doesn't really describe what it does and sounds more like a `boolean` value.